pkgs/net-lib/net/sendurl.rkt

```diff
@@ -203,6 +203,8 @@
     (define json
       (with-handlers ([exn:fail? (λ (exn) (fail (exn-message exn)))])
         (read-json in)))
+    (when (eof-object? json)
+      (fail "didn't find LSHandlers key in (empty) json"))
```

Comment:
Why not say something about `plutil`? Like "failed to get a json from `plutil` got eof-object? instead"